sklearn/datasets/_samples_generator.py

```diff
@@ -2110,6 +2110,14 @@ def make_biclusters(
         words using bipartite spectral graph partitioning. In Proceedings
         of the seventh ACM SIGKDD international conference on Knowledge
         discovery and data mining (pp. 269-274). ACM.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import make_biclusters
+    >>> shape = (100, 50)
+    >>> data, rows, columns = make_biclusters(shape=shape,n_clusters=5,random_state=42)
```

Comment:
```suggestion
    >>> data, rows, columns = make_biclusters(shape=shape, n_clusters=5, random_state=42)
```
just a formatting tweak.

What do you think of putting the shape directly in the function call instead of assigning it to a variable first? would somehow make it more uniform with the other arguments.